Gem/Code/Source/Automation/PrecommitWizardSettings.h

```diff
@@ -47,32 +52,46 @@ namespace AtomSampleViewer
         // currently set to track the ScriptReport index and the ScreenshotTestInfo index.
         using ReportIndex = AZStd::pair<size_t, size_t>;
 
-        void GenerateReportsToInpsect(const AZStd::vector<ScriptReporter::ScriptReport>& scriptReports)
+        // This function does the following:
+        // 1. Collect passing screenshot tests and sorts them by decreasing diff score.
+        // 2. Collect failed screenshot tests and sorts them by decreasing diff score. 
+        void ProcessScriptReports(const AZStd::vector<ScriptReporter::ScriptReport>& scriptReports)
         {
             m_reportsOrderedByThresholdToInspect.clear();
+            m_failedReports.clear();
 
             for (size_t i = 0; i < scriptReports.size(); ++i)
             {
                 const AZStd::vector<ScriptReporter::ScreenshotTestInfo>& screenshotTestInfos = scriptReports[i].m_screenshotTests;
                 for (size_t j = 0; j < screenshotTestInfos.size(); ++j)
                 {
-                    if (screenshotTestInfos[j].m_officialComparisonResult.m_resultCode == ScriptReporter::ImageComparisonResult::ResultCode::ThresholdExceeded)
+                    // Collect and sort reports that passed by threshold. This will be used to detect false positives
+                    // e.g. a test is reported to pass by being below the threshold when in fact it's simply because the threshold is too
+                    // high
```

Comment:
```suggestion
                    // Collect and sort reports that passed by threshold. This will be used to detect false negatives
                    // e.g. a test is reported to pass by being below the threshold when in fact it's simply because the threshold is too
                    // high
```

Reply:
"False positive" is when the test fails but it shouldn't.